src/toil/wdl/wdltoil.py

```diff
@@ -142,6 +142,184 @@ def decorated(*args: Any, **kwargs: Any) -> Any:
         return cast(F, decorated)
     return decorator
 
+def remove_common_leading_whitespace(expression: WDL.Expr.String, tolerate_blanks: bool = True, tolerate_dedents: bool = False, tolerate_all_whitespace: bool = True, debug: bool = False) -> WDL.Expr.String:
```

Comment:
Could there be a test for this function?

Reply:
I've added some tests.